packages/nodejs/src/diagnose.ts

```diff
@@ -93,15 +93,16 @@ export class DiagnoseTool {
 
     // we want to fall over if this value isn't present
     // (it should be)
-    const logPath = this.#config.data.logPath!
+    const logDirPath = this.#config.data.logPath!
+    const logPath = logDirPath + "/appsignal.log"
 
     // add any paths we want to check to this object!
     const files = {
       working_dir: {
         path: process.cwd()
       },
       log_dir_path: {
-        path: logPath.replace("/appsignal.log", "")
```

Comment:
Removing this line could break with existing configurations. Our docs now say you need to specify the whole path: https://docs.appsignal.com/nodejs/configuration/options.html#option-logpath
Note to us: update that.

If someone specifies the "appsignal.log" file, the output would become `/tmp/appsignal.log/appsignal.log` (note the "appsignal.log" is mentioned twice).

Maybe we should keep this behavior (for a couple versions), but move it up to the other line changes in this PR where we fetch the `logPath` config value?

Reply:
I’ll add a test case for this. 👍